commands/load_balancers.go

```diff
@@ -92,6 +92,14 @@ With the load-balancer command, you can list, create, or delete load balancers,
 		"A comma-separated list of ALLOW rules for the load balancer, e.g.: `ip:1.2.3.4,cidr:1.2.0.0/16`")
 	AddStringSliceFlag(cmdLoadBalancerCreate, doctl.ArgDenyList, "", []string{},
 		"A comma-separated list of DENY rules for the load balancer, e.g.: `ip:1.2.3.4,cidr:1.2.0.0/16`")
+	AddStringFlag(cmdLoadBalancerCreate, doctl.ArgLoadBalancerDomains, "", "",
+		"A comma-separated list of domains required to ingress traffic to a global load balancer")
+	AddStringFlag(cmdLoadBalancerCreate, doctl.ArgGlobalLoadBalancerSettings, "", "",
+		"Global load balancer settings")
+	AddStringFlag(cmdLoadBalancerCreate, doctl.ArgGlobalLoadBalancerCDNSettings, "", "",
+		"Global load balancer CDN settings")
```

Comment:
Is there any more detail you can provide on the format that these settings should be in? This help output doesn't really explain to the customer how they can use this flag.

Reply:
ACK, will add example config string to the helper text to make it more explicit.